Gems/Atom/Feature/Common/Code/Source/CoreLights/CapsuleLightFeatureProcessor.cpp

```diff
@@ -207,5 +231,20 @@ namespace AZ
             return m_lightBufferHandler.GetElementCount();
         }
 
+        void CapsuleLightFeatureProcessor::UpdateBounds(LightHandle handle)
```

Comment:
I considered that, but it would add a bunch of checks for something that doesn't actually happen that much or cost that much. It's all just some very basic math that's only done when a light is changed.